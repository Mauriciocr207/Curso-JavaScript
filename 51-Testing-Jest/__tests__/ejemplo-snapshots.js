const user = {
    name: 'Juan 2',
    balance: 500,
    type: "premium",
}

describe('client', () => {
    test('is juan', () => expect(user).toMatchSnapshot())
})