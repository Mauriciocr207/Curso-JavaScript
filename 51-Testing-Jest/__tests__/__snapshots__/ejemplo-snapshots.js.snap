// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`client is juan 1`] = `
{
  "balance": 500,
  "name": "Juan 2",
  "type": "premium",
}
`;
